packages/server/src/comment/comment.model.ts

```diff
@@ -0,0 +1,38 @@
+import { Directive, Field, ID, ObjectType } from '@nestjs/graphql';
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import mongoose, { Document } from 'mongoose';
+
+@Schema()
+@ObjectType()
+@Directive('@key(fields: "_id")')
+export class Comment {
+  @Field(() => ID)
+  _id: mongoose.Types.ObjectId;
+
+  @Prop({ required: true })
+  @Field()
+  fileId: string;
```

Comment:
For GraphQl the type should be File

Reply:
Do you mean I also need to resolve this field in resolver?

Reply:
Yep, so like user this can be resolved to the full file